.env.kesaseteli.example

```diff
@@ -15,6 +15,7 @@ OIDC_RP_CLIENT_ID=
 OIDC_RP_CLIENT_SECRET=
 LOGIN_REDIRECT_URL=https://localhost:3000/
 LOGIN_REDIRECT_URL_FAILURE=https://localhost:3000/login?error=true
+LOGOUT_REDIRECT_URL=https://localhost:3000/login?logout=true
```

Comment:
add this to `.env.tet.example` too?

Reply:
Oh I forgot to remove that, apparently only backend needs that variable, oh and we don't have example files for backend?

Reply:
Oh they are in same file, I will fix now